### PR TITLE
Elisp updates

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

Motivated by the melpification of enlight.

Most recent Hydra job:
- ~~https://hydra.nix-community.org/eval/8059~~
- https://hydra.nix-community.org/eval/9676
  - https://hydra.nix-community.org/eval/9676
  - https://github.com/nix-community/emacs-overlay/commit/5e9c97365e40a9a7f576dba8516eae5fd4f0fc62

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by jian-lin** [comment]
Could you add a link of the related hydra job?  That link provide many info, such as the commit of emacs-overlay and the number of failed packages, for reviewing.

**Comment by AndersonTorres** [comment]
How can I obtain this link?

**Comment by jian-lin** [comment]
https://hydra.nix-community.org/jobset/emacs-overlay/master lists some jobs, choose one matching the commit of your update.

**Comment by AndersonTorres** [comment]
@jian-lin done.

**Comment by jian-lin** [comment]
Is this ready for review?  I ask because it is in draft mode.

**Comment by AndersonTorres** [comment]
> Is this ready for review? I ask because it is in draft mode.

Yes.

File: pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix [hunk 1]
```diff
@@ -15,8 +15,6 @@ in
 
   agda2-mode = callPackage ./manual-packages/agda2-mode { };
 
-  beancount = callPackage ./manual-packages/beancount { };
```
Comment (jian-lin) [inline pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix:18]: I cannot find `beancount` on GNU ELPA or GNU-devel ELPA.

**Review by jian-lin — COMMENTED** [review]
> Most recent Hydra job: https://hydra.nix-community.org/eval/8059

I cannot reproduce the same diff using `update-from-overlay` with commit `76e9df7e3f701466fb91591a985f2e3ef73cf65b`.  Is the link correct?

File: pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix:18]: It is on NonGNU ELPA: https://elpa.nongnu.org/nongnu/beancount.html

https://github.com/NixOS/nixpkgs/pull/325055/files#diff-feb07e0cf8a541b8fcc5b0a76d87bd9d37cdc81709e778f9de14370f4be69542R215

**Comment by AndersonTorres** [comment]
> > Most recent Hydra job: https://hydra.nix-community.org/eval/8059
> 
> I cannot reproduce the same diff using `update-from-overlay` with commit `76e9df7e3f701466fb91591a985f2e3ef73cf65b`. Is the link correct?

Strange. I merely updated using the script and picked the latest commit from emacs-overlay + latest job from hydra link.

I reran the script right now.

(To be more precise, I run the script in Master branch and cherry-picked it over Staging.)

**Review by jian-lin — APPROVED** [review]

**Pull request merged**